.gitignore

```diff
@@ -166,4 +166,5 @@ settings.json
 **/*.pkl
 **/*.png
 **/*.pt
+!assets/checkpoints_for_examples/model_merge_example/*.pt
```

Comment:
I think this is okay to do the pattern ignore here, since the pt files won't change too much. Just wanted to note (in case you didn't know) that you can override ignores for files by doing a `git commit -f` with a path to the specific file. Those will then be pushed to the remote, but any changes to them will remain untracked in the future.

Reply:
Good to know! In that case, I will just remove it :)